doc/userguide/rules/datasets.rst

```diff
@@ -183,6 +227,10 @@ md5
   in the file as hex encoded string
 sha256
   in the file as hex encoded string
+ipv4
+  in the file as string
+ip
+  in the file as string, it can be IPv6 or IPv4 address (in standard notation or in IPv4 in IPv6 one)
```

Comment:
Should the last portion be something like: "in standard notation or in IPv4 or IPv6 ones"?

Reply:
Porposing: (standard notation or IPv4 in IPv6 one)

Reply:
Sounds good, makes it easier to understand :P